kerl

```diff
@@ -1007,9 +1100,9 @@ _do_build() {
         # shellcheck disable=SC2086  # Double quote to prevent globbing and word splitting
         if ! log_build_cmd "_flags ./otp_build configure $KERL_CONFIGURE_OPTIONS"; then
             show_build_logfile "configure failed."
-            autoclean "$2"
             list_remove builds "$1 $2"
-            exit 1
+            autoclean "$2"
+            return 1
```

Comment:
Many of these were "just" moved around for consistency. We `autoclean` just before we return.